cachi2/core/package_managers/npm.py

```diff
@@ -124,23 +107,16 @@ def resolved_url(self) -> Optional[str]:
             # v2+ file dependency (or a workspace)
             elif self.path:
                 return f"file:{self.path}"
```

Comment:
No need to check `if self.path` anymore. 

`bool(self.path)` == the package is from a v2+ lockfile (which is always true now)